python/ql/src/Security/CWE-798/HardcodedCredentials.ql

```diff
@@ -3,7 +3,7 @@
  * @description Credentials are hard coded in the source code of the application.
  * @kind problem
  * @problem.severity error
- * @precision medium
+ * @precision high
```

Comment:
This is going to change the behaviour of this query on LGTM. Please add a change note.